childController/rectangle.py

```diff
@@ -0,0 +1,19 @@
+import sys
+sys.path.append("C:/Users/Payal Chaudhari/Desktop/Python new folder/Practise_folder/Practice_automation/baseController")
+
+from baseClass import Base
+
+
+class Rectangle(Base):
+    def __init__(self,length,breadth):
+        self.length= length
```

Comment:
Please declare the variable even on top as **private** access specifier and initialize with 0, then on `__init__` assign the user value, that will follow proper **OOP** principles then